mods/ra/maps/ant-01/rules.yaml

```diff
@@ -0,0 +1,128 @@
+Player:
+	PlayerResources:
+		DefaultCash: 1500
+World:
+	ScriptLobbyDropdown@difficulty:
+		ID: difficulty
+		Label: Difficulty
+		Values:
+			easy: Easy
+			normal: Normal
+			hard: Hard
+		Default: easy
+	LuaScript:
+		Scripts: ant-01.lua, ant-attack.lua
+	MissionData:
+		BackgroundVideo: antintro.vqa
+		Briefing: We've lost contact with one of our outposts. Before it went off-line, we recieved a brief communique about giant ants. We're unsure what to make of this report, so we want you to investigate.  \n\nScout the area, bring the outpost back on-line, and report your findings. If there is a threat, reinforcements will be sent in to help you.  \n\nKeep the base functional and radio contact open -- we don't want to lose the outpost again.
+
+^Palettes:
+	IndexedPlayerPalette@scoutant:
+		BaseName: scoutant
+		BasePalette: player
+		RemapIndex: 80, 81, 82, 83, 84, 85, 86, 87, 88, 89, 90, 91, 92, 93, 94, 95
+		PlayerIndex:
+			AntMan: 133, 134, 135, 136, 137, 138, 139, 140, 141, 141, 142, 142, 143, 143, 143, 143
```

Comment:
Those are the wrong colors, the original used
> 112, 113, 114, 115, 116, 117, 118, 119, 120, 121, 122, 123, 123, 123, 123, 123

Well, not sure about the exact distribution, I don't have the original palette.cps here, but definitely those 12 colors.